app/controllers/registrations_controller.rb

```diff
@@ -1,7 +1,9 @@
 class RegistrationsController < Devise::RegistrationsController
 
   def new
-    super { |user| user.managers.build.build_nonprofit }
```

Comment:
I didn't mean to leave this change in - at first I was trying to create a new Social Profile when the new user is created (i.e. user.build_social_profile), but I don't think this works if nothing is actually entered upon sign up. I changed this back and committed it.